WooCommerce/Classes/ViewRelated/Orders/Order Details/OrderDetailsViewController.swift

```diff
@@ -115,10 +113,6 @@ final class OrderDetailsViewController: UIViewController {
         super.viewDidLayoutSubviews()
         tableView.updateHeaderHeight()
     }
-
-    override var shouldShowOfflineBanner: Bool {
-        !isSplitViewInOrdersTabEnabled
-    }
```

Comment:
I'm not sure if there are other concerns, but the offline banner seems to work correctly when we turn off/on the connection if we override with `true` here:
```
override var shouldShowOfflineBanner: Bool {
    true
}
```
![Simulator Screenshot - iPad Pro (12 9-inch) (6th generation) - 2024-02-27 at 12 01 09](https://github.com/woocommerce/woocommerce-ios/assets/3812076/c62460db-5379-449f-a9b7-b54351724a8d)